dotnet/src/SemanticKernel.UnitTests/Contents/ChatMessageContentTests.cs

```diff
@@ -68,10 +70,23 @@ public void ContentPropertySetterShouldUpdateContentOfFirstTextContentItem()
 
         var sut = new ChatMessageContent(AuthorRole.User, items: items)
         {
-            Content = "fake-content-1-update"
+            Content = content
+        };
+
+        Assert.Equal(content, ((TextContent)sut.Items[1]).Text);
+    }
+
+    [Fact]
+    public void ContentPropertySetterShouldNotAddTextContentToItemsCollection()
+    {
+        // Arrange
+        var sut = new ChatMessageContent(AuthorRole.User, content: null)
+        {
+            Content = null
         };
 
-        Assert.Equal("fake-content-1-update", ((TextContent)sut.Items[1]).Text);
+        // Assert
+        Assert.Empty(sut.Items);
```

Comment:
Can you also add the Assert on the Content property when the items are empty (what is expected for the `Content`)